extensions/vault/runtime/src/main/java/io/quarkus/vault/pki/ConfigCRLOptions.java

```diff
@@ -0,0 +1,18 @@
+package io.quarkus.vault.pki;
+
+/**
+ * Options for configuration the CRL
```

Comment:
_Options for CRL configuration_ or _Options for configuring the CRL_

Reply:
👍